lib/git-view.coffee

```diff
@@ -117,12 +123,16 @@ class GitView extends HTMLElement
       if ahead > 0
         @commitsAhead.textContent = ahead
         @commitsAhead.style.display = ''
+        @commitsAheadTooltipDisposable?.dispose()
+        @commitsAheadTooltipDisposable = atom.tooltips.add @commitsAhead, title: "#{ahead} commits ahead of upstream"
```

Comment:
This should probably use something like `_.pluralize` so that it doesn't end up as `1 commits`